### DISCUSSION TITLE
Discord Testnet Faucet Bot

### DISCUSSION THREAD

**Discussion by fotescodev** [onflow/flow#1227] [Developer Experience]
Would love to see a discord faucet similar to what other chains do. Increases community engagement and it's simple.

**Comment by jacob-tucker** [comment]
Hey! What do you mean by this? I could probably incorporate this into Emerald bot.

**Reply by LanfordCai** [reply]
A bot with `/faucet address` command for users to get testnet coins in Discord, I think

**Reply by justinbarry** [reply]
Correct. The Kovan on Ethereum has a similar thing. See: https://gitter.im/kovan-testnet/faucet?at=5dd685a4c4fca14de3c3be4a

**Reply by jacob-tucker** [reply]
Ahhh that would be awesome! If we had the ability to mint flow token I could easily incorporate into emerald bot, but we would need flow team’s approval

**Reply by justinbarry** [reply]
Why would you need the teams approval?

**Reply by jacob-tucker** [reply]
I dont have the ability to mint flow testnet tokens. Would need the resource in the cadence  to do that
